app/assets/javascripts/discourse/tests/helpers/create-store.js

```diff
@@ -65,7 +65,7 @@ export default function (customLookup = () => {}) {
             this._topicListAdapter || TopicListAdapter.create({ owner: this });
           return this._topicListAdapter;
         }
-        if (type === "key-value-store:main") {
+        if (type === "service:key-value-store") {
```

Comment:
Doesn't this need to support both forms for back compat? Or is it used just in core tests?

Reply:
I can only find one occurence of `createStore` in `all-the-*`, and that plugin doesn't make use of the key-value-store. So I think we should be ok.